fortifyapi/client.py

```diff
@@ -234,53 +239,53 @@ def create(self, project_name, version_name, project_id=None, description="Creat
         :return: Returns the Version object
         :rtype: fortifyapi.Version
         """
-        with self._api as api:
-            r = api.post(f"/api/v1/projectVersions", {
-                'name': version_name,
-                'description': description,
-                'active': active,
-                'committed': committed,
-                'project': {
-                    'id': project_id,  # if this is None it will create the project
-                    'name': project_name,
+        # Test if Project Version exists
+        if self.versions.test(project_name, version_name):
+            exit("Project: {} Version: {} exists!".format(project_name, version_name))
```

Comment:
throw an exception, people could use this anywhere and you do not want a process exiting.

Reply:
The self.versions.test endpoint is a bool.  Therefore, the create is either we create true or not false.  if we don't exit on an existing Project/Version the func returns a NoneType.

Reply:
raise an exception, do not exit. Exit is a syscode and will terminate a process. You want to be able to catch it.